### PR TITLE
Track nodes drop off for test state

### LINKED ISSUE #70: Too slow transition (>10 min) to FINISHED/TERMINATED state.
It takes 10 minutes to transition over to FINISHED/TERMINATED state.

Please change the state transition trigger to enter FINISHED/TERMINATED state if **>=10%** nodes disappear between polls to node-finder.

2019-06-25T16:53:36.721324147Z [OkHttp http://node_finder:8080/...] INFO org.radixdlt.explorer - Received nodes: 1186
2019-06-25T16:54:06.719749113Z [OkHttp http://node_finder:8080/...] INFO org.radixdlt.explorer - Received nodes: 1186
2019-06-25T16:54:36.736987851Z [OkHttp http://node_finder:8080/...] INFO org.radixdlt.explorer - Received nodes: 1164
2019-06-25T16:55:06.721880403Z [OkHttp http://node_finder:8080/...] INFO org.radixdlt.explorer - Received nodes: 1006
2019-06-25T16:55:36.723853963Z [OkHttp http://node_finder:8080/...] INFO org.radixdlt.explorer - Received nodes: 989
2019-06-25T16:56:06.719518121Z [OkHttp http://node_finder:8080/...] INFO org.radixdlt.explorer - Received nodes: 988
2019-06-25T16:56:36.719610481Z [OkHttp http://node_finder:8080/...] INFO org.radixdlt.explorer - Received nodes: 988
2019-06-25T16:57:06.718539549Z [OkHttp http://node_finder:8080/...] INFO org.radixdlt.explorer - Received nodes: 873
2019-06-25T16:57:36.716913668Z [OkHttp http://node_finder:8080/...] INFO org.radixdlt.explorer - Received nodes: 553
2019-06-25T16:58:06.715046985Z [OkHttp http://node_finder:8080/...] INFO org.radixdlt.explorer - Received nodes: 353
2019-06-25T16:58:36.713666162Z [OkHttp http://node_finder:8080/...] INFO org.radixdlt.explorer - Received nodes: 203
2019-06-25T16:59:06.713693636Z [OkHttp http://node_finder:8080/...] INFO org.radixdlt.explorer - Received nodes: 125
2019-06-25T16:59:36.713053926Z [OkHttp http://node_finder:8080/...] INFO org.radixdlt.explorer - Received nodes: 76
2019-06-25T17:00:06.712311114Z [OkHttp http://node_finder:8080/...] INFO org.radixdlt.explorer - Received nodes: 56
2019-06-25T17:00:36.713861420Z [OkHttp http://node_finder:8080/...] INFO org.radixdlt.explorer - Received nodes: 32
2019-06-25T17:01:06.713101938Z [OkHttp http://node_finder:8080/...] INFO org.radixdlt.explorer - Received nodes: 23
2019-06-25T17:01:36.712686536Z [OkHttp http://node_finder:8080/...] INFO org.radixdlt.explorer - Received nodes: 14
2019-06-25T17:02:06.712779228Z [OkHttp http://node_finder:8080/...] INFO org.radixdlt.explorer - Received nodes: 9
2019-06-25T17:02:36.712499545Z [OkHttp http://node_finder:8080/...] INFO org.radixdlt.explorer - Received nodes: 4
2019-06-25T17:03:06.712278227Z [OkHttp http://node_finder:8080/...] INFO org.radixdlt.explorer - Received nodes: 4
2019-06-25T17:03:36.712952347Z [OkHttp http://node_finder:8080/...] INFO org.radixdlt.explorer - Received nodes: 2
2019-06-25T17:04:06.712976510Z [OkHttp http://node_finder:8080/...] INFO org.radixdlt.explorer - Received nodes: 2
2019-06-25T17:04:36.712327950Z [OkHttp http://node_finder:8080/...] INFO org.radixdlt.explorer - Received nodes: 1
2019-06-25T17:05:06.711993750Z [OkHttp http://node_finder:8080/...] INFO org.radixdlt.explorer - Received nodes: 1
2019-06-25T17:05:36.712898995Z [OkHttp http://node_finder:8080/...] INFO org.radixdlt.explorer - Received nodes: 1

### PR DESCRIPTION
**Pull request by laszlourszuly**
There is now a configuration property that controls how many nodes are allowed to "drop off" compared to the peak count. If more nodes are dropped off than what is considered acceptable, the
`TestStateProvider` reserves the right to consider the test eligible for a state transition.

Fixes: #70

### CODE REVIEW

**Review by blenessy — APPROVED** [review]
LGTM

**Comment by blenessy** [comment]
@swiec please test this on staging network and approve PR if working

File: explorer/service/src/main/java/org/radixdlt/explorer/system/TestStateProvider.java [hunk 1]
```diff
@@ -222,9 +222,9 @@ private boolean isMeasuring() {
         // TODO: Piers wants to transition to FINISHED state when >=90% of the data has been
         // validated (storedPerShard). Problem is that systemInfo is not normalised like in
         // MetricsProvide.calculateMetrics.      
-        if (currentState == TestState.STARTED) {
-            return (currentState.getStartTimestamp() + 60 * 60 * 1000) > System.currentTimeMillis();
-        }
+        //if (currentState == TestState.STARTED) {
+        //    return (currentState.getStartTimestamp() + 60 * 60 * 1000) > System.currentTimeMillis();
+        //}
```
Comment (laszlourszuly) [inline explorer/service/src/main/java/org/radixdlt/explorer/system/TestStateProvider.java]: Just want to double check @blenessy: Is it okay, in your opinion, to remove this code? (If so, I'll go ahead and actually delete the lines as well).

Comment (blenessy) [reply explorer/service/src/main/java/org/radixdlt/explorer/system/TestStateProvider.java]: Yes I think so. Killing the network is a better (more dynamic) way of signalling test FINISHED to the Explorer.

**Review by blenessy — APPROVED** [review]

**Review by swiec — COMMENTED** [review]
Tested on integration branch. Please merge in into master.

**Review by swiec — APPROVED** [review]
Ready to go.

**Pull request merged**